<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>241</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ipCon">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nomeUser"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Porta:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="portaCon">
       <property name="text">
        <string>9999</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect!</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mensagem:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mensagem"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Destino:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="destinoLista"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>610</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>821</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
